sdk/keyvault/azure-security-keyvault-keys/src/cryptography/cryptography_client.cpp

```diff
@@ -75,108 +102,86 @@ CryptographyClient::CryptographyClient(
 }
 
 Azure::Response<DecryptResult> CryptographyClient::Decrypt(
     DecryptParameters const& parameters,
     Azure::Core::Context const& context)
 {
-  return m_pipeline->SendRequest<DecryptResult>(
-      context,
-      Azure::Core::Http::HttpMethod::Post,
-      [&parameters]() {
-        return DecryptParametersSerializer::DecryptParametersSerialize(parameters);
-      },
-      [&parameters](Azure::Core::Http::RawResponse const& rawResponse) {
-        auto result = DecryptResultSerializer::DecryptResultDeserialize(rawResponse);
-        result.Algorithm = parameters.Algorithm;
-        return result;
-      },
-      {"decrypt"});
+  // Send and parse respone
+  auto rawResponse = SendCryptoRequest(
+      {"decrypt"}, DecryptParametersSerializer::DecryptParametersSerialize(parameters), context);
+  auto value = DecryptResultSerializer::DecryptResultDeserialize(*rawResponse);
+  value.Algorithm = parameters.Algorithm;
+  return Azure::Response<DecryptResult>(std::move(value), std::move(rawResponse));
 }
 
 Azure::Response<WrapResult> CryptographyClient::WrapKey(
     KeyWrapAlgorithm algorithm,
     std::vector<uint8_t> const& key,
     Azure::Core::Context const& context)
 {
-  return m_pipeline->SendRequest<WrapResult>(
-      context,
-      Azure::Core::Http::HttpMethod::Post,
-      [&algorithm, &key]() {
-        return KeyWrapParametersSerializer::KeyWrapParametersSerialize(
-            KeyWrapParameters(algorithm.ToString(), key));
-      },
-      [&algorithm](Azure::Core::Http::RawResponse const& rawResponse) {
-        auto result = WrapResultSerializer::WrapResultDeserialize(rawResponse);
-        result.Algorithm = algorithm;
-        return result;
-      },
-      {"wrapKey"});
+  // Send and parse respone
+  auto rawResponse = SendCryptoRequest(
+      {"wrapKey"},
+      KeyWrapParametersSerializer::KeyWrapParametersSerialize(
+          KeyWrapParameters(algorithm.ToString(), key)),
+      context);
+  auto value = WrapResultSerializer::WrapResultDeserialize(*rawResponse);
+  value.Algorithm = algorithm;
+  return Azure::Response<WrapResult>(std::move(value), std::move(rawResponse));
 }
 
 Azure::Response<UnwrapResult> CryptographyClient::UnwrapKey(
     KeyWrapAlgorithm algorithm,
     std::vector<uint8_t> const& encryptedKey,
     Azure::Core::Context const& context)
 {
-  return m_pipeline->SendRequest<UnwrapResult>(
-      context,
-      Azure::Core::Http::HttpMethod::Post,
-      [&algorithm, &encryptedKey]() {
-        return KeyWrapParametersSerializer::KeyWrapParametersSerialize(
-            KeyWrapParameters(algorithm.ToString(), encryptedKey));
-      },
-      [&algorithm](Azure::Core::Http::RawResponse const& rawResponse) {
-        auto result = UnwrapResultSerializer::UnwrapResultDeserialize(rawResponse);
-        result.Algorithm = algorithm;
-        return result;
-      },
-      {"unwrapKey"});
+  // Send and parse respone
+  auto rawResponse = SendCryptoRequest(
+      {"unwrapKey"},
```

Comment:
move these strings to constants

Reply:
it's only used once. But I can do it, sure.

Reply:
Personally, I don't mind the hard-coded strings being inline. But, its personal choice.

Reply:
Usually I would create a constant when the string is used from more than one place only.  Otherwise, I would just leave it inline.